### PR TITLE
Server cleanup and simplification

### PR DESCRIPTION
**Pull request by bryevdv**

### CODE REVIEW

**Comment by hhuuggoo** [comment]
anything involving storing/loading callbacks can also be removed - that was a feature we played with but ended up not using at all

**Comment by bryevdv** [comment]
@hhuuggoo thanks! That is good to know

**Comment by bryevdv** [comment]
@hhuuggoo is there any reason `UnathorizedException` can't go into `bokeh.exceptions` or failing that, `bokeh.server.exceptions`?

**Comment by bryevdv** [comment]
@hhuuggoo what about `views/plugins.py` and `jsgenerate`? I think those are not currently used? We want to make it possible to extend the models collection but I think it would take more work than those do and I'd rather start afresh if they are not needed now.

**Comment by hhuuggoo** [comment]
jsgenerate was used as a crufty way of auto-generating javascript classes that mirror new python classes (you have to define what js class it inherits from) - It's used in some of the applet stuff.   I'm sure there's a better way to do that, but that's what it does right now

**Comment by hhuuggoo** [comment]
pretty sure the exception can be moved anywhere

**Comment by bryevdv** [comment]
@hhuuggoo perfect, will keep it around for now.

**Comment by bryevdv** [comment]
@mattpap @fpliger @hhuuggoo @damianavila I'd like to merge this onto new "dev 0.10" branch ASAP. There's lots of other things I want to do as smaller more focused PRs but they all depend on getting this stuff out of the way. Please take a look, and leave comments. Given the longer timeline of 0.10 dev I think merging it early into the branch is ok even if there are some things to fix up later.

File: bokeh/server/templates/tests/tab_play.html [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    var resizeTimer;
-    clearTimeout(resizeTimer);
-    resizeTimer = setTimeout(resizeRoot, 500);
-  });
-
-  // Init root pane size.
-  resizeRoot();
-</script>
-
-{% endblock %}
```
Comment (fpliger) [inline bokeh/server/templates/tests/tab_play.html]: out of curiosity. what was that for?

Comment (bryevdv) [inline bokeh/server/templates/tests/tab_play.html]: I have no idea. :) But I couldn't find any reference to it being used anywhere. I think maybe it was an old test harness. We can replace it if I am mistaken about it not being needed, though.

**Pull request closed without merging**